cmd/micro/micro.go

```diff
@@ -151,6 +152,10 @@ func main() {
 
 	var err error
 
+	if err = puffy.PledgePromises("stdio rpath wpath cpath tty proc exec"); err != nil {
```

Comment:
did you forget tmppath?

thanks. have a nice day. keep up the good work.

[ret394](https://github.com/ret394)